File: Java/프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.java
class Solution {
    public int solution(String my_string, String is_prefix) {
        if(my_string.length()<is_prefix.length()){
            return 0;
        }
        if (my_string.substring(0,is_prefix.length()).equals(is_prefix)){
            return 1;
        }
        return 0;
    }
}

File: C#/프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.cs
using System;

public class Solution {
    public int solution(string my_string, string is_prefix) {
        int answer = 0;
        if (my_string.Length<is_prefix.Length){
            return 0;
        }
        if (my_string.Substring(0,is_prefix.Length).Equals(is_prefix)){
            return 1;
        }
        return 0;
    }
}